src/config.js

```diff
@@ -34,6 +34,14 @@ export const flags = {
   },
 };
 
+export const settings = {
+  prettyName: {
+    name: 'Pretty Name',
+    description: 'Prettify the kedro node name',
```

Comment:
'Prettify' sounds ambiguous - I would reword this as something along the lines of ' display a more read-able version of the node name'